charts/frontend/templates/services-deployment.yaml

```diff
@@ -50,6 +50,14 @@ spec:
           - name: shell-keys
             mountPath: /etc/ssh/keys
           {{- end }}
+        lifecycle:
+        {{- if $service.lifecycle }}
```

Comment:
Let's add support for an optional lifecycle hook but let's not set a default. So please move this if-statement above the `lifecycle:` line and remove the else-statement.

Reply:
Done. It wasn't clear to me if I should leave `lifecycle` "hardcoded" inside the if block or rely on "passing" it from values.yaml but at least it's added in the same way as the other hooks in this file.